Sources/ArcGISToolkit/Components/Form/FormInputs/SingleLineTextInput.swift

```diff
@@ -62,10 +71,13 @@ struct SingleLineTextInput: View {
         }
         .formTextInputStyle()
         TextInputFooter(
-            currentLength: text.count,
+            text: text,
             isFocused: isFocused,
             element: element,
-            input: input
+            input: input,
+            rangeDomain: rangeDomain,
+            isNumeric: isNumeric,
+            isDecimal: isDecimal
```

Comment:
Is there a better way to do this instead of passing in both values? What about moving the `FieldType` extension containing the `isNumeric` and `isDecimal` properties to its own file and just passing in the field type to the footer? That would eliminate one argument. Thoughts?